### DISCUSSION TITLE
为什么 js 文件里无法使用process.env.xxxx

### DISCUSSION THREAD

**Discussion by J1angyue** [NervJS/taro#14254] [Q & A]
为什么 js 文件里无法使用process.env.xxxx??????????????????????
taro版本：3.6.8
vue 文件里可以 访问 process.env.xxxx，js文件里访问process.env.xxxx，然后vue文件import 这个js 文件就会报错：ReferenceError: process is not defined

复现仓库：https://github.com/J1angyue/taro-process-undefined.git

**Comment by yoyo837** [comment]
如果运行时是node.js是有的, 如果是webpack5打包后就没有

**Reply by J1angyue** [reply]
可是 vue 和 js里同时是nodejs运行时，vue文件内可以访问，js文件就访问不了
